public/app/features/search/hooks/useDashboardSearch.ts

```diff
@@ -31,7 +32,11 @@ export const useDashboardSearch = (query: DashboardQuery, onCloseSearch: () => v
           if (selectedItem.type === DashboardSearchItemType.DashFolder) {
             onToggleSection(selectedItem as DashboardSection);
           } else {
-            getLocationSrv().update({ path: selectedItem.url });
+            getLocationSrv().update({
+              path: selectedItem.url.startsWith(config.appSubUrl)
+                ? selectedItem.url.split(config.appSubUrl + '/')[1]
+                : selectedItem.url,
```

Comment:
think you can use `locationUtil.assureBaseUrl` function from grafana/data for this :)

Reply:
I remember that you mentioned it before, but I couldn't find it!